dts/arm/st/f0/stm32f072X8.dtsi

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2018 Linaro Limited
```

Comment:
Please use your own Copyright.
Even if this is a copy/paste you're the one doing the upstream and (hopefully) validation effort.

Reply:
done!